### PR TITLE
browser: move  impl of savecomments command to Toolbar dispatch

### PR DESCRIPTION
**Pull request by meven**
Change-Id: Ifcb34977b72bb0eccd23667deb4750637810e569

* Resolves: # 
* Target version: master 

### Summary

cc #8029

### CODE REVIEW

File: browser/src/control/Toolbar.js [hunk 1]
```diff
@@ -1153,6 +1153,14 @@ L.Map.include({
 			this.sendUnoCommand('.uno:DeletePrintArea');
 			this.print();
 			break;
+		case 'savecomments':
+			if (this._map.isPermissionEditForComments()) {
```
Comment (eszkadev) [inline browser/src/control/Toolbar.js:1157]: did you test this code? does it work?

I think `this` here is `map` object already

Comment (meven) [reply browser/src/control/Toolbar.js]: I put this back in draft, it wasn't tested yet.
This is fixed now.

**Comment by meven** [comment]
Do you know where I can test this @eszkadev?

I am surprised at how simple the change is, does this patch look as expected?

**Comment by eszkadev** [comment]
https://github.com/CollaboraOnline/online/blob/543256280133e0a5d46bebc623796020425dd72a/browser/src/control/Control.Menubar.js#L543

you can find it in the menubar in classic mode in Impress/Draw

**Comment by meven** [comment]
> https://github.com/CollaboraOnline/online/blob/543256280133e0a5d46bebc623796020425dd72a/browser/src/control/Control.Menubar.js#L543
> 
> you can find it in the menubar in classic mode in Impress/Draw

Only in readOnly mode though.

I commented the part that need it to be read-only and added a break in Toolbar.js,
I can confirm the code is called, and works as before.

**Comment by meven** [comment]
> > https://github.com/CollaboraOnline/online/blob/543256280133e0a5d46bebc623796020425dd72a/browser/src/control/Control.Menubar.js#L543
> > 
> > you can find it in the menubar in classic mode in Impress/Draw
> 
> Only in readOnly mode though.
> 
> I commented the part that need it to be read-only and added a break in Toolbar.js, I can confirm the code is called, and works as before.

I can confirm with "&permission=readonly" too.

**Review by eszkadev — APPROVED** [review]

**Pull request merged**